zilencer/views.py

```diff
@@ -106,3 +114,66 @@ def remote_server_notify_push(request,  # type: HttpRequest
                                      badge=1, zulip=apns_payload)
 
     return json_success()
+
+def register_remote_server(request):
+    # type: (HttpRequest) -> HttpResponse
+    if request.method == 'POST':
+        form = RemoteServerRegistrationForm(request.POST)
+        if form.is_valid():
+            email = form.cleaned_data['email']
+            api_key = form.cleaned_data['api_key']
+            uuid = form.cleaned_data['uuid']
+            hostname = form.cleaned_data['hostname']
+            with transaction.atomic():
```

Comment:
Just saw Steve's comment, so maybe this is fine. For `send_email` at least though it would be helpful to compactify everything, since it makes it easier find and make wholesale changes to the set of emails.